src/system-health/tests/test_system_health.py

```diff
@@ -0,0 +1,224 @@
+"""
+    Unit test cases for system health checker. The current test case contains:
+        1. test_user_defined_checker mocks the output of a user defined checker and verify class UserDefinedChecker
+        2. test_service_checker mocks the output of monit service and verify class ServiceChecker
+        3. test_hardware_checker mocks the hardware status data in db and verify class HardwareChecker
+    And there are class that are not covered by unit test. These class will be covered by sonic-mgmt regression test.
+        1. HealthDaemon
+        2. HealthCheckerManager
+        3. Config
+"""
+import os
+import sys
+
+from .mock_connector import MockConnector
+import swsssdk
+
+swsssdk.SonicV2Connector = MockConnector
+from mock import Mock, MagicMock, patch
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+sys.path.insert(0, modules_path)
+from health_checker.manager import HealthCheckerManager
+from sonic_py_common import device_info
```

Comment:
Do the last two imports need to come after the path setup? If not, can you move to the top and group/organize imports per PEP8 standards?